### PR TITLE
multiple file upload listener

### PR DESCRIPTION
**Pull request by JarekW**

### CODE REVIEW

**Comment by JarekW** [comment]
@fsi-open
also maybe someone know how to test this in behat

**Comment by JarekW** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1: is it still WIP?

File: Resources/doc/multiple_file_upload.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Multiple file upload
+
+In order to use multiple file upload feature you must add a listener to collection holding file forms. When instantiating listener pass name of field that holds file instance.
+
+```php
+$formBuilder->get('files')->addEventSubscriber(new MultiUploadCollectionListener('file'));
```
Comment (chives) [inline Resources/doc/multiple_file_upload.md]: It's no longer necessary to add subscriber. Now it's enough to set ``multi_upload_field`` option on collection form.

**Comment by JarekW** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
can i move php 7.0 to `allow_failures` section because of phpspec?

**Comment by chives** [comment]
@JarekW as for now - yes, but specs should be converted to docblocks instead of type hints before 7.0 release

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

File: Form/TypeExtension/MultiUploadCollectionExtension.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     */
+    public function getExtendedType()
+    {
+        return 'collection';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
```
Comment (rn0) [inline Form/TypeExtension/MultiUploadCollectionExtension.php]: you should add `configureOptions` method for compatibility with future version of symfony

**Comment by rn0** [comment]
:+1:

**Pull request merged**